daprdocs/content/en/operations/components/setup-bindings/supported-bindings/_index.md

```diff
@@ -9,56 +9,56 @@ no_list: true
 
 ### Generic
 
-| Name | Input<br>Binding | Output<br>Binding | Status |
-|------|:----------------:|:-----------------:|--------|
-| [Apple Push Notifications (APN)]({{< ref apns.md >}}) |  | ✅ | Alpha |
-| [Cron (Scheduler)]({{< ref cron.md >}}) | ✅ | ✅ | Alpha |
-| [HTTP]({{< ref http.md >}})           |    | ✅ | Alpha |
-| [InfluxDB]({{< ref influxdb.md >}})       |    | ✅ | Alpha |
-| [Kafka]({{< ref kafka.md >}})         | ✅ | ✅ | Alpha |
-| [Kubernetes Events]({{< ref "kubernetes-binding.md" >}}) | ✅ |    | Alpha |
-| [MQTT]({{< ref mqtt.md >}})           | ✅ | ✅ | Alpha |
-| [MySQL]({{< ref mysql.md >}})       |    | ✅ | Alpha |
-| [PostgreSql]({{< ref postgres.md >}})       |    | ✅ | Alpha |
-| [Postmark]({{< ref postmark.md >}})       |    | ✅ | Alpha |
-| [RabbitMQ]({{< ref rabbitmq.md >}})   | ✅ | ✅ | Alpha |
-| [Redis]({{< ref redis.md >}})         |    | ✅ | Alpha |
-| [SMTP]({{< ref smtp.md >}})         |    | ✅ | Alpha |
-| [Twilio]({{< ref twilio.md >}})       |    | ✅ | Alpha |
-| [Twitter]({{< ref twitter.md >}})       | ✅ | ✅ | Alpha |
-| [SendGrid]({{< ref sendgrid.md >}})       |    | ✅ | Alpha |
+| Name | Input<br>Binding | Output<br>Binding | Version | Status |
```

Comment:
I think there may be some confusion here between Version and Status. My initital question is why is something v1.0 when its Status is alpha? Seems like we should only go to v1 when Status is Stable or GA. We may also be able to remove the status column if we move all alpha rows to v0.1 and Stable to v1.0

Reply:
The version and GA status are independent. Components have a only MAJOR version number so there is no minor version. Just v1 - This is an indication of an particular implementation. The GA status indicates that this has has conformance tests and customer usage. See the PR from @mukundansundar on conformance requirements

Reply:
Ohhh gothca. That makes much more sense. Can we make the column header a link to the new docs page on component versioning?